src/Pheetup/SiteBundle/Controller/DefaultController.php

```diff
@@ -41,16 +42,19 @@ public function groupAction(Group $group)
     }
 
     /**
-     * @Route("/event/{id}")
-     * @ParamConverter("event",class="PheetupMeetupBundle:Event")
+     * @Route("/{domain}/join")
+     * @ParamConverter("group",class="PheetupUserBundle:Group")
      */
-    public function registerGroupAction(Event $event, $id)
+    public function joinAction(Group $group)
     {
-        $em = $this->getDoctrine()->getEntityManager();
-        $group = $em->getRepository('PheetupUserBundle:Group')->find($id);
-        $event->setGroup($group);
-        $em->persist($event);
-        $em->flush();
+        $isLogin = $this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_FULLY');
+        if ($isLogin) {
+            $token = $this->get('security.token_storage')->getToken();
+            $user = $token->getUser();
+            $user->addGroup($group);
+        }
```

Comment:
Buraya bir de giriş yapmadıysa durumu lazım. Ama önce BaseController yapmalıyız bu getUser gibi metodlar için.